src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -16,33 +16,244 @@
 package com.google.cloud.compute.v1.it;
 
 import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
```

Comment:
Please use Truth.assertThat

Reply:
> Please use Truth.assertThat

why i used thruth ? if it is done using junit.

Reply:
Because, all new test cases follows Truth and truth is more readable compared to Assert or TestCase

Reply:
done